datamap/lucene/src/main/java/org/apache/carbondata/datamap/lucene/LuceneFineGrainDataMapFactory.java

```diff
@@ -57,7 +57,7 @@ public LuceneFineGrainDataMapFactory(CarbonTable carbonTable, DataMapSchema data
           DataMapWriter.getDefaultDataMapPath(tableIdentifier.getTablePath(),
               segment.getSegmentNo(), dataMapName), segment.getConfiguration()));
     } catch (MemoryException e) {
-      LOGGER.error("failed to get lucene datamap , detail is {}" + e.getMessage());
+      LOGGER.error("failed to get lucene datamap, detail is {}" + e.getMessage());
```

Comment:
```suggestion
      LOGGER.error(String.format("failed to get lucene datamap, detail is %s", e.getMessage()));
```
I think this log can't print the exception message

Reply:
+1, it should can print the exception message, but String.format is better.

Reply:
fixed